Sources/SwiftInspectorVisitors/PropertySyntaxVisitor.swift

```diff
@@ -102,6 +92,19 @@ public final class PropertySyntaxVisitor: SyntaxVisitor {
 
     return modifier
   }
+
+  private func findTypeName(from node: TypeAnnotationSyntax?) -> String? {
+    guard
```

Comment:
This is basically a copy/paste from https://github.com/fdiaz/SwiftInspector/pull/99/files#diff-8622b0ddc6698d01f425675d628fce88a76b661f30c567a9756e1282c9981e87L38